pgx/src/iterators.rs

```diff
@@ -0,0 +1,129 @@
+use pgx_utils::sql_entity_graph::metadata::{
+    ArgumentError, ReturnVariant, ReturnVariantError, SqlTranslatable, SqlVariant,
+};
+
+use crate::{pg_sys, IntoDatum};
+
+pub struct SetOfIterator<'a, T>
+where
+    T: std::panic::UnwindSafe + std::panic::RefUnwindSafe,
```

Comment:
We don't need these paths everywhere.

```suggestion
    T: UnwindSafe + RefUnwindSafe,
```